paparazzi/paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PaparazziPlugin.kt

```diff
@@ -186,8 +187,12 @@ class PaparazziPlugin : Plugin<Project> {
         // why not a lambda?  See: https://docs.gradle.org/7.2/userguide/validation_problems.html#implementation_unknown
         test.doLast(object : Action<Task> {
           override fun execute(t: Task) {
-            val uri = reportOutputDir.get().asFile.toPath().resolve("index.html").toUri()
-            test.logger.log(LIFECYCLE, "See the Paparazzi report at: $uri")
+            val path = reportOutputDir.get().asFile.toPath().resolve("index.html")
+            if (path.exists()) {
+              test.logger.log(LIFECYCLE, "See the Paparazzi report at: ${path.toUri()}")
+            } else {
+              test.logger.log(LIFECYCLE, "Paparazzi failure report not generated.")
+            }
```

Comment:
Can you elaborate more what was the executed scenario that did not result in a report being generated?

Reply:
When there was no failure that file will not exist at that path

Reply:
I would say in that case just don't output anything? When unit tests pass there's no output either.

Reply:
> When there was no failure that file will not exist at that path

This isn't a failure path, but rather a report that should always be generated.  Did you mean, when there *is* a failure?